src/content/structured/components/divider/accessibility.mdx

```diff
@@ -0,0 +1,49 @@
+---
+path: "/components/divider/accessibility"
+
+date: "2024-12-02"
+
+title: "Divider"
+
+status: "BETA"
+
+subTitle: "Dividers are used to separate different elements in groups of content. Visually, they look like rules or lines."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/divider" },
+    { title: "Code", path: "/components/divider/code" },
+    { title: "Accessibility", path: "/components/divider/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+Horizontal dividers use the semantic `<hr>` ('thematic break' or 'horizontal rule') HTML element.
```

Comment:
```suggestion
Horizontal dividers without a label use the semantic `<hr>` ('thematic break' or 'horizontal rule') HTML element.
```
There may be a better way to explain this

Reply:
Can't really think of a better way so have just gone with this 👍